src/standard-live-connect.js

```diff
@@ -103,17 +104,20 @@ function _processArgs (args, pixelClient, enrichedState) {
  * @return {StandardLiveConnect|null}
  * @private
  */
-function _getInitializedLiveConnect (liveConnectConfig) {
+function _getInitializedLiveConnect (liveConnectConfig, initializationCallback) {
   try {
     if (window && window.liQ && window.liQ.ready) {
-      const mismatchedConfig = window.liQ.config && _configMatcher(window.liQ.config, liveConnectConfig)
-      if (mismatchedConfig) {
-        const error = new Error()
-        error.name = 'ConfigSent'
-        error.message = 'Additional configuration received'
-        emitter.error('LCDuplication', JSON.stringify(mismatchedConfig), error)
+      if ((!qualifiedConfig(liveConnectConfig)) ||
```

Comment:
We could also go with another option. When we detect we have 2 instances we can shut down the previous instance, unregistering all listeners.
We also can create a new topic which we publish to whenever we are starting a new instance

Reply:
> We could also go with another option. When we detect we have 2 instances we can shut down the previous instance, unregistering all listeners.

We will lose external listeners that way. To be able to keep them, we would need to be able to distinguish them from the internal ones. Which is doable, but maybe not worth the overhead.

Reply:
It should not be too difficult. We would keep using the same global bus, just have a topic to publish cancellation commands to. The external consumer would not notice anything